src/routers/webhooks.py

```
@@ -0,0 +1,91 @@
+    """
+    Delete a specific webhook endpoint.
+    """
+
+    if not jwt_params.ad:
+        raise AuthenticationException("Unauthorized access to resource")
+
+    await crud.delete_webhook_endpoint(db, endpoint_id)
+
+
+@router.post("/test-emit", response_model=None)
+async def test_emit(
```

Ruff (0.12.2)

Missing return type annotation for public function `test_emit`

Add return type annotation: `None`

(ANN201)